### ISSUE TITLE
change event not triggered on drag & drop

### ISSUE THREAD

**Issue by tstamac** [markusslima/bootstrap-filestyle#163]
When you drag and drop files to the component change event is not fired.
Tried on Chrome and Firefox.

in 
```javascript
on('drop', 
```
after (line 316)
```javascript
   $('[name="filedrag"]').css('z-index', '-1'); 
```
there should be
```javascript
   _self.options.onChange(files);
```
This doesn't check if files dropped are actually different from previous state, but at least it fires the event.
